tests/unit/introspection.tcl

```diff
@@ -233,6 +233,27 @@ start_server {tags {"introspection"}} {
         r client list
     } {*name= *}
 
+    test {Coverage: Basic CLIENT CACHING} {
+        set rd_redirection [redis_deferring_client]
+        $rd_redirection client id
+        set redir_id [$rd_redirection read]
+        r CLIENT TRACKING on OPTIN REDIRECT $redir_id
+        r CLIENT CACHING yes
+        r CLIENT TRACKING off
+    } {OK}
```

Comment:
if we execute each of these commands without doing anything, let's at least also validate the response in the tcl test.
and maybe make sure the test title explains what we do (although maybe the word "coverage" is enough, specifically if it's repeated in other similar tests..